biscuit-auth/src/datalog/mod.rs

```diff
@@ -625,7 +626,9 @@ impl World {
 
             index += 1;
             if index == limits.max_iterations {
-                break Err(Execution::RunLimit( crate::error::RunLimit::TooManyIterations));
+                break Err(Execution::RunLimit(
```

Comment:
formatting issue ?

Reply:
yeah, there is a commit fixing formatting errors